doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc

```diff
@@ -0,0 +1,82 @@
+[id='environment-immutable-server-kjar-proc']
+// definitions for RH-SSO include
```

Comment:
I think you can turn these steps into prereqs:
You can extract this file from the rhpam-7.2.0-openshift-templates.zip product deliverable file. You can download the file from the Software Downloads page.

Prerequisite
You have downloaded the BLAH file from the Software Downloads page and extracted it.

Reply:
These procedures need a big reorg including this issue, post 7.2.